markdownlint/TOP010_useLazyNumbering/TOP010_useLazyNumbering.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  names: ["TOP010", "ordered-list-for-lazy-numbering"],
+  description: "Ordered lists should use lazy numbering",
```

Comment:
2 things:
1. As with the above comment, a little more elaboration should make it clearer what the rule actually expects of someone (may not know what lazy numbering alone is, and even then some rules enforce `0.` prefixing).
2. Use "must" instead of "should" given it's an enforced rule.
```suggestion
  description: "Ordered lists must always use 1. as a prefix (lazy numbering)",
```